aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -37,11 +42,15 @@
 import software.amazon.awssdk.services.rds.model.RebootDbInstanceRequest;
 import software.amazon.awssdk.services.rds.model.RemoveRoleFromDbInstanceRequest;
 import software.amazon.awssdk.services.rds.model.RestoreDbInstanceFromDbSnapshotRequest;
+import software.amazon.awssdk.services.rds.model.RestoreDbInstanceToPointInTimeRequest;
 import software.amazon.awssdk.utils.StringUtils;
+import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
 import software.amazon.rds.common.handler.Tagging;
 import software.amazon.rds.dbinstance.util.ResourceModelHelper;
 
 public class Translator {
+    public static String RESTORE_TIME_PATTERN = "yyyy-MM-dd HH:mm:ss";
```

Comment:
I see that RDS API accepts ISO timestamps, like `2009-09-07T23:45:00Z`, which would fail this pattern validation.